portfolio/src/main/java/com/google/sps/utils/Constants.java

```diff
@@ -0,0 +1,8 @@
+package com.google.sps.utils;
+
+public class Constants {
+    public final static String COMMENT_KEY = "Comment";
```

Comment:
If it's public it needs javadoc to describe the use?

Reply:
Fixed